components/Form/index.js

```diff
@@ -14,9 +14,11 @@ export default function Form({ setNewProjects, projects }) {
 
     const formData = new FormData(event.target);
     const newProject = Object.fromEntries(formData);
-    newProject.id = `${projects.length + 1}`;
+    const idOfLatestProject = Number(projects[projects.length - 1].id);
+    newProject.id = `${idOfLatestProject + 1}`;

```

Comment:
At a first glance, this seems like a good solution for ensuring that IDs are generated continuously from the last project ID 🙌. Nice! 
This should prevent the potential issue of assigning the same ID again when projects are deleted from the array. It requires that the projects array is not empty, though. Here's a potential solution:

`const idOfLatestProject = Number(projects[projects.length - 1]?.id || 0);`